src/test/java/de/blackforestsolutions/dravelopsdatamodel/JourneyTest.java

```diff
@@ -0,0 +1,54 @@
+package de.blackforestsolutions.dravelopsdatamodel;
+
+import org.assertj.core.api.WritableAssertionInfo;
+import org.assertj.core.internal.Strings;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.regex.Pattern;
+
+import static de.blackforestsolutions.dravelopsdatamodel.objectmothers.JourneyObjectMother.getJourneyBuilderWithNoEmptyFields;
+import static de.blackforestsolutions.dravelopsdatamodel.objectmothers.JourneyObjectMother.getJourneyWithEmptyFields;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class JourneyTest {
+
+    @Test
+    void test_build_journey_generates_conform_sha1Id() throws IOException {
+        Pattern sha1IdPattern = Pattern.compile("[0-9a-f]{40}");
```

Comment:
We use this regex here which is ok but we should comment it to be sure or shouldnt we ? 
For me it reads out the following way:

All small written letters and numbers are allowed and it was to be 40 sings long.

Is my reading correct ?

Reply:
Done